util/fetch_deps.ts

```diff
@@ -0,0 +1,244 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Is it possible to not disable this? Or, were you planning to follow up with a different PR to change this?

Reply:
Oh, I forget to push commit that this has been removed